src/main/java/seedu/address/model/exercise/ExerciseDate.java

```diff
@@ -14,7 +14,8 @@
 public class ExerciseDate {
 
     public static final String MESSAGE_CONSTRAINTS =
-        "Date input should be in the format DD-MM-YYYY and it should not be blank";
+        "Date input should be in the format DD-MM-YYYY and should not be blank. "
+        + "It should only be from within one year before to the current date.";
```

Comment:
Kinda awkward phrasing. How about `The accepted range of dates is today to one year before today.`? And maybe add an `eg. 18-05-2020`